hadoop-ozone/dev-support/checks/junit.sh

```diff
@@ -56,7 +57,7 @@ if [[ ${ITERATIONS} -gt 1 ]] && [[ ${OZONE_REPO_CACHED} == "false" ]]; then
   mvn ${MAVEN_OPTIONS} -DskipTests install
 fi
 
-REPORT_DIR=${OUTPUT_DIR:-"$DIR/../../../target/${CHECK}"}
+REPORT_DIR=${OUTPUT_DIR:-"$DIR/../../../${TARGET_DIR}/${CHECK}"}
```

Comment:
I don't think we need another variable (`TARGET_DIR`), `flaky-test-check` workflow can set `OUTPUT_DIR` directly.